### PR TITLE
Add support for ios11 and ios12

### PR DESCRIPTION
**Pull request by zltnDC**

### CODE REVIEW

File: packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h
Comment (bartekpacia) [inline packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h:1]: What's up with changes in this file? Did some whitespace got changed?

**Comment by bartekpacia** [comment]
overall LGTM

File: packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h
Comment (zltnDC) [reply packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h:1]: There are three changes If I remember correctly:
1. [server startAndReturnError:err];
2. listDartTestsWithCompletionHandler -> listDartTestsWithCompletion
3. runDartTestWithName: completionHandler -> completion

Comment (zltnDC) [reply packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h:1]: And clang-format changed whitespaces.

**Review by bartekpacia — APPROVED** [review]
Would be great if the macro diff was smaller, otherwise OK

File: packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h
Comment (zltnDC) [reply packages/patrol/ios/Classes/PatrolIntegrationTestRunner.h:1]: The second point causes this change. 
[appServiceClient listDartTestsWithCompletion] is now formatted to a single line.

**Comment by bartekpacia** [comment]
To merge before native selectors rework.

**Pull request merged**